parser/src/ast/boundary_constraints.rs

```diff
@@ -36,8 +36,8 @@ impl BoundaryConstraint {
     }
 
     /// Returns a clone of the constraint's value expression.
```

Comment:
nit: This comment should be changed if we no longer return a clone of the value.

Reply:
Great catch, thanks!